src/main/flight/mixer.c

```diff
@@ -826,19 +922,9 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs) {
 #endif
     loggingThrottle = throttle;
     motorMixRange = motorMixMax - motorMixMin;
```

Comment:
propably would better to include motorMixRange calc to applyMixerClipAdjustment func?